judge/models/contest.py

```diff
@@ -139,6 +142,8 @@ def can_see_scoreboard(self, user):
             return True
         if user.is_authenticated and self.organizers.filter(id=user.profile.id).exists():
             return True
+        if self.see_contest_scoreboard.filter(id=user.profile.id).exists():
```

Comment:
This needs the `user.is_authenticated` check as the line above, or we'll crash on an anonymous user who has no profile.